README.rst

```diff
@@ -53,7 +53,7 @@ the initial render to take at least an hour, possibly more. (Since Minecraft
 maps are practically infinite, the maximum time this could take is also
 infinite!)
 
-If you press ctrl-C, it will stop. The next run will pick up where it left off.
+If you press <kbd>Ctrl-C</kbd>, it will stop. The next run will pick up where it left off.
```

Comment:
It appears that google's markdown renderer doesn't support `<kbd>`.

Can you switch this to something else, maybe backticks?